extensions/ql-vscode/src/databases/db-item.ts

```diff
@@ -11,6 +13,25 @@ export enum DbItemKind {
 
 export interface RootLocalDbItem {
   kind: DbItemKind.RootLocal;
+  children: LocalDbItem[];
+}
+
+export type LocalDbItem =
+  | LocalListDbItem
+  | LocalDatabaseDbItem;
+
+export interface LocalListDbItem {
+  kind: DbItemKind.LocalList;
+  listName: string;
+  databases: LocalDatabaseDbItem[];
+}
+
+export interface LocalDatabaseDbItem {
```

Comment:
Didn't abbreviate "database" to "db" here, since we're referring to the `databases` property of the config 🤔 (also, `LocalDbDbItem` would probably be weird 🤷🏽 )